lib/fortnox/api/repositories/base/savers.rb

```diff
@@ -9,19 +9,30 @@ def save( entity )
           hash = entity_to_hash( entity )
           json = hash.to_json
 
-          response = entity.new? ? save_new( json ) : update_existing( json )
+          entity.new? ? save_new( json ) : update_existing( hash )
```

Comment:
You mean like this:

```
hash = entity_to_hash( entity )

return save_new( hash ) if entity.new?
update_existing( hash )
```

And then `save_new` converts to json as in your next comment?